src/README.md

```diff
@@ -17,3 +17,57 @@ Almost all components of RisingWave are developed in rust, and they are split to
 [state-store]: https://github.com/risingwavelabs/risingwave/blob/main/docs/state-store-overview.md
 [meta-service]: https://github.com/risingwavelabs/risingwave/blob/main/docs/meta-service.md
 [risedev]: https://github.com/risingwavelabs/risingwave/tree/main/src/risedevtool
+
+## Understanding RisingWave Macros
+
+RisingWave uses a lot of macros to simplify development.
+You may choose the read to macro definition directly,
+but something complementary / easier is to use `cargo expand`
```

Comment:
even more handy is https://rust-analyzer.github.io/manual.html#expand-macro-recursively

<img width="1352" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/ecd113d3-4cc3-4d21-a04b-89d5ca9150bd">

Reply:
Thanks!